eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
ok, sure if the format is different, it makes sense, pls remove eagle-server.sh.
i could not find the eagle.properties file in this commit or in the project so i couldnt check on it.